packages/protocol/test-sol/unit/governance/network/Governance.t.sol

```diff
@@ -263,6 +263,13 @@ contract GovernanceTest is Test, TestConstants, Utils {
   }
 }
 
+contract TransitionToL2AfterL1 is GovernanceTest {
```

Comment:
Use the new WhenL2 util contract, as in [Validators test](https://github.com/celo-org/celo-monorepo/blob/release/core-contracts/12/packages/protocol/test-sol/unit/governance/validators/Validators.t.sol#L434)

Reply:
implemented in https://github.com/celo-org/celo-monorepo/pull/11267/commits/d00851155545fe3e6fb51f515a9ca3efcf1a925e